twilight-model/src/poll/media.rs

```diff
@@ -0,0 +1,71 @@
+use crate::id::{marker::EmojiMarker, Id};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct PollMedia {
+    /// The emoji of the field.
+    ///
+    /// When creating a poll answer with an emoji, one only
+    /// needs to send either the id (custom emoji) or name
+    /// (default emoji) as the only field.
+    pub emoji: Option<PartialPollMediaEmoji>,
+    /// The text of the field.
+    pub text: Option<String>,
+}
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct PartialPollMediaEmoji {
```

Comment:
This is the same as `twilight_model::gateway::presence::ActivityEmoji`.

I think it is okay to keep them seperate, but maybe these should be unified at some point.